config/services/resolver.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use CommerceWeavers\SyliusTpayPlugin\Resolver\BlikAliasResolver;
+use CommerceWeavers\SyliusTpayPlugin\Resolver\BlikAliasResolverInterface;
+
+return static function(ContainerConfigurator $container): void {
+    $services = $container->services();
+
+    $services->set('commerce_weavers_sylius_tpay.resolver.blik_alias', BlikAliasResolver::class)
+        ->args([
+            service('commerce_weavers_sylius_tpay.repository.blik_alias')->nullOnInvalid(),
```

Comment:
When can it be nulled?

Reply:
Oops, that must be some leftover. Already removed.

Reply:
Oops, that must be some leftover. Already removed.